File: app/src/main/java/android1601/itstep/org/kidsgame/program/KidsApplication.java
package android1601.itstep.org.kidsgame.program;

import android.app.Application;
import android.view.View;

import com.raizlabs.android.dbflow.config.FlowConfig;
import com.raizlabs.android.dbflow.config.FlowManager;

/**
 * Created by roman on 14.03.2017.
 */

public class KidsApplication extends Application {
    private static final String TAG = KidsApplication.class.getName();
    private static KidsApplication instance;

    public static KidsApplication getInstance() {
        return instance;
    }

    @Override
    public void onCreate() {
        instance = this;
        super.onCreate();
        FlowManager.init(new FlowConfig.Builder(this).build());

    }
}

File: app/src/main/java/android1601/itstep/org/kidsgame/program/KidsApplication.kt
package android1601.itstep.org.kidsgame.program

import android.app.Application
import android.view.View

import com.raizlabs.android.dbflow.config.FlowConfig
import com.raizlabs.android.dbflow.config.FlowManager

/**
 * Created by roman on 14.03.2017.
 */

class KidsApplication : Application() {

    override fun onCreate() {
        instance = this
        super.onCreate()
        FlowManager.init(FlowConfig.Builder(this).build())

    }

    companion object {
        private val TAG = KidsApplication::class.java.name
        var instance: KidsApplication? = null
            private set
    }
}
